### DISCUSSION TITLE
Key decision points

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/CA_quillback_rockfish_2025#7] [General]
Based on my experience with the 2021 assessment, these elements will be important decision points for this cycle

- Additional recreational or commercial fleets (see #6 for details)
- Sexes **combined** or ~separate~ (see #28 for details)
- Estimate growth within the model or fix outside the model. If estimate, use CAAL or not
- Treatment of discards (e.g. **add in catches**, ~add as discard fleet~) (see #35 for some details, other issues are linked there)
- Maximum age (what value to use - see #37 for details)
- Sum to zero constraint for recdevs (would affect CA model in 2021. Would it affect oregon model?)
- Treatment of closed areas if at all (see #65 for details)

**Comment by melissamonk-NOAA** [comment]
I look at the Albin et al. Table 1 data available [here](https://github.com/pfmc-assessments/california-data/tree/main/recreational-fishery).  The estimates of quillback rockfish in 1,000s of fish by district from 1981-1986 are small with large CVs.  The estimates from 1983-1986 could be used to partition the catch in these years. However, Mendocino and Sonoma county are grouped together.
